### PR TITLE
7522 pa11y tests and some sequence tests

### PR DESCRIPTION
**Pull request by mrinsin**

### CODE REVIEW

File: web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        caseOrder: [{ ...caseOrderItemWithoutNote, calendarNote: 'delete me' }],
+        trialSessionId: mockTrialSessionId,
+      },
+    ]);
+  });
+
+  it('should update state.trialSessions after deleting the note on the specified trialSession', async () => {
+    await test.runSequence('deleteCalendarNoteSequence');
+
+    expect(test.getState('trialSessions')).toEqual([
```
Comment (rachelschneiderman) [inline web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js:48]: Nit: Can an expectation be added that the calendarNote is no longer present on that trial session?

File: web-client/src/presenter/sequences/openAddEditCaseNoteModalSequence.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { CerebralTest } from 'cerebral/test';
```
Comment (rachelschneiderman) [inline web-client/src/presenter/sequences/openAddEditCaseNoteModalSequence.test.js:1]: 👏  👏  👏  sequence tests! Nice!

**Review by rachelschneiderman — APPROVED** [review]

File: web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    applicationContext
+      .getUseCases()
+      .saveCalendarNoteInteractor.mockReturnValue({
+        trialSessionId: mockTrialSessionId,
+      });
+
+    test.setState('caseDetail', {
+      docketNumber: mockDocketNumber,
+      trialsSessionId: mockTrialSessionId,
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js]: Nit: typo `trialsSessionId`

File: web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js (hunk 1 above)
Comment (mrinsin) [reply web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js:48]: Absolutely!

Comment (mrinsin) [reply web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js:48]: Hmm there is some weirdness with this test.. down to pair later today?

Comment (rachelschneiderman) [reply web-client/src/presenter/sequences/deleteCalendarNoteSequence.test.js:48]: Yes!

**Pull request merged**